cli/glide.yaml

```diff
@@ -1,7 +1,9 @@
-package: github.com/apache/brooklyn-client
+package: github.com/apache/brooklyn-client/cli
 import:
 - package: github.com/urfave/cli
+# using git@github.com:johnmccabe/jsonpath.git temporarily as original repo deleted March 2018
```

Comment:
Didn't realise comments were supported in the `glide.yaml`, apart from that it looks 100%